WalletWasabi.Backend/Controllers/BlockchainController.cs

```diff
@@ -291,8 +291,7 @@ public IActionResult GetFilters([FromQuery, Required]string bestKnownBlockHash,
 				return NoContent();
 			}
 
-			var response = new FiltersResponse
-			{
+			var response = new FiltersResponse {
```

Comment:
I misconfigured CodeMaid, because I most definitely don't want this Java-like brackets.

Reply:
Fixed. I didn't find how to properly configure that in CodeMaid. But there is a way to fix it through .editorconfig. I opened #1868 to fix that.